util/json.go

```diff
@@ -65,7 +70,7 @@ func MustUnmarshalJSON(bs []byte) interface{} {
 // If the data cannot be encoded, this function will panic. This function is for
 // test purposes.
 func MustMarshalJSON(x interface{}) []byte {
-	bs, err := json.Marshal(x)
+	bs, err := MarshalJSON(x)
```

Comment:
Do we need the util changes? 👀

Reply:
bb61934fc9959e0db8fe76ec53ac7066156c9a49

Reply:
I removed `MarshalJSON` completely